content/zh-cn/docs/tasks/debug/debug-application/determine-reason-pod-failure.md

```diff
@@ -65,56 +69,72 @@ the container starts.
    写入 `/dev/termination-log` 文件。
    容器写完 "Sleep expired" 消息后就终止了。
 
-<!-- 1. Display information about the Pod: -->
-1. 显示 Pod 的信息：
+<!--
+1. Display information about the Pod:
+-->
+2. 显示 Pod 的信息：
 
-        kubectl get pod termination-demo
+   ```shell
+   kubectl get pod termination-demo
+   ```
 
-   <!--Repeat the preceding command until the Pod is no longer running.-->
+   <!--
+   Repeat the preceding command until the Pod is no longer running.
+   -->
    重复前面的命令直到 Pod 不再运行。
 
-<!-- 1. Display detailed information about the Pod: -->
-1. 显示 Pod 的详细信息：
+<!--
+1. Display detailed information about the Pod:
+-->
+3. 显示 Pod 的详细信息：
```

Comment:
Just keep using `1.` for all list item, it would be better for version history.

Reply:
For English, keep using `1. ` is fine. For Chinese localization, the comment lines inserted in between will break the numbering, so explicit numbering is needed.